### ISSUE TITLE
Why does synchronization stop intermittently?

### ISSUE THREAD

**Issue by junghyun0783** [ethereum/go-ethereum#30502]
sepolia testnet sync stops intermittently, and I don't know the cause.
It runs normally, but it keeps stopping intermittently, and when I turn it off and on, the sync starts again.

The causes I guess are
1. Hardware problem
2. config setting problem
I guess the above two...

I leave open the possibility that it is a hardware problem.
Spec.
os: windows
cpu: Xeon(R) CPU E5-2620
memory: 64G
hard: 2TB (no ssd)
What could be the problem?

this is my commands
geth : `cmd /k geth -syncmode "snap" --http.api admin,eth,debug,net,txpool,personal,web3,engine -datadir ".\dataSepolia" -http --http.port 20201 --http.addr 0.0.0.0 --nat "any" --rpc.allow-unprotected-txs --snapshot=false -ipcdisable --au thrpc.jwtsecret ../consensus/jwt_sepolia.hex --authrpc.vhosts="*" --authrpc.addr=0.0.0.0 --lightkdf --cache 16384`

sepolia prysm : `prysm.bat beacon-chain --execution-endpoint=http://localhost:8551 --sepolia --datadir=./prysm_data_sepolia --jwt-secret=../jwt_sepolia.hex --rpc-host=0.0.0.0 --grpc-gateway-host=0.0.0.0 --genesis-state=sepolia_genesis.ssz --suggested-fee-recipient=0x67CDB8344FFCa7aaaeda9e7a4F4f1A9 cb4D1FC64`

plz help me!

**Comment by jwasinger** [comment]
Can you post a snippet from the logs when it stops syncing?

**Comment by junghyun0783** [comment]
@jwasinger 
I can't check the log right now...
I think this problem will occur again soon.

Is there a problem with the geth or prysm commands?

**Comment by jwasinger** [comment]
`--au thrpc.jwtsecret` well this should prevent geth from starting up.  But given that you said it was syncing, it sounds like it started up correctly (maybe you pasted the command incorrectly).

**Comment by junghyun0783** [comment]
@jwasinger 
Yes yes, it's a typo when you "paste" now.
Currently, it's working normally by entering "--authrpc.jwtsecret".
You're saying there's no problem with the command....
Then, does that mean there's no problem with the hardware?

**Comment by jwasinger** [comment]
I'm saying that if the node is syncing, that is a good indication that you have set it up correctly.

I have no clue what the underlying issue is, and logs would be a good start to try and diagnose what is going on..

**Comment by junghyun0783** [comment]
[
![66](https://github.com/user-attachments/assets/7953fac4-2e1d-4c4d-a1e5-8aacc8e487cc)
](url)
I checked this morning and it's stopped again. I'll show you the log first.

**Comment by holiman** [comment]
@junghyun0783 it looks like your CL client is having peer-problems: `Waiting for enough suitable peers before syncing`, which causes it to stop feeding EL (geth) with consensus updates. This leads to "Execution client not syncing", because geth doesn't know what to sync to. 

TLDR; seems to me that the root (this time, at least), is that CL can't find sufficient peers.

**Comment by holiman** [comment]
See the first item here: https://docs.prylabs.network/docs/troubleshooting/issues-errors#troubleshooting-your-beacon-node , and follow those steps. Closing this, since it's not a geth-issue.
